### DISCUSSION TITLE
Does not export blender 2.93.6 textures to Character Creator 3

### DISCUSSION THREAD

**Discussion by Domi94sosa** [soupday/cc3_blender_tools#36] [Q&A] [answered]
Does not export blender 2.93.6 textures to Character Creator 3
I select the Export Accesory option and the only thing I can get is the object without any type of texture. Nor do I have a folder where to find the textures to place manually.
https://ibb.co/mz34ddx

**Comment by soupday** [comment, marked as answer]
The FBX import accessory in CC3, doesn't bring in any textures, just the mesh.

You'll need to bake or produce the textures then load them into the material slots in CC3.
